src/test/java/org/fogbowcloud/manager/core/models/linkedlists/SynchronizedDoublyLinkedListTest.java

```diff
@@ -24,99 +24,133 @@ public void initialize() {
 
     @Test
     public void testAddFirst() {
+        // verify
         assertNull(this.list.getHead());
         assertNull(this.list.getCurrent());
         assertNull(this.list.getTail());
```

Comment:
Although it is not in the template, it was defined that all Assert must also not be static.

Reply:
Nice catch, will fix this.